library/core/tests/slice.rs

```diff
@@ -2504,3 +2504,19 @@ fn test_slice_from_ptr_range() {
         assert_eq!(slice::from_ptr_range(range), &arr);
     }
 }
+
+#[cfg(not(bootstrap))]
+#[test]
+#[should_panic = "slice len overflow"]
+fn test_flatten_size_overflow() {
+    let x = &[[(); usize::MAX]; 2][..];
+    let _ = x.flatten();
+}
+
+#[cfg(not(bootstrap))]
+#[test]
+#[should_panic = "slice len overflow"]
+fn test_flatten_mut_size_overflow() {
+    let mut x = &mut [[(); usize::MAX]; 2][..];
```

Comment:
Looks like this one needs to just be
```suggestion
    let x = &mut [[(); usize::MAX]; 2][..];
```

@Cyborus04 Can you apply that, squash again, and push?  (Since you seem to be around 🙃)